expression/builtin_json_test.go

```diff
@@ -55,9 +56,13 @@ func (s *testEvaluatorSuite) TestJSONUnquote(c *C) {
 		Expected interface{}
 	}{
 		{nil, nil},
+		{``, ``},
+		{`""`, ``},
+		{`''`, ``},
 		{`"a"`, `a`},
 		{`3`, `3`},
-		{`{"a": "b"}`, `{"a":"b"}`},
+		{`{"a": "b"}`, `{"a": "b"}`},
+		{`{"a":     "b"}`, `{"a":     "b"}`},
 		{`"hello,\"quoted string\",world"`, `hello,"quoted string",world`},
```

Comment:
since we remove prefix and suffix '"' and '\'' in this PR,
why can this case run successfully before?

Reply:
Priviously when call json_unquote on `String`, we **parse** it. That will remove prefix and suffix `"` when the string is a valid JSON.